ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -1608,3 +1608,60 @@
                 default-excludes: true
                 fingerprint: false
                 only-if-success: false
+      - 'cloud-{name}-{test_name}':
+          disabled: false
+          triggers: []
```

Comment:
Is this Jenkins job triggered by GitHub comments or a timer?

Reply:
Given that this is a relatively resource-intensive pipeline, triggering it via comments or scheduled tasks may not be ideal. Would it be more appropriate for the project maintainers to trigger it as needed? cc @antoninbas @tnqn

Reply:
I'll let @luolanzone comment as well but maybe we could the following:
- run the test _weekly_ - of course it's very important for resource cleanup to _always_ work consistently as we don't want to leak EC2 instances
- trigger the job manually from the Jenkins UI prior to merging a PR which impacts secondary network support, and potentially prior to releasing a new Antrea version

Reply:
if there is no way to have an allow-list for users, let's do it from the UI

@XinShuYang will we be able to set up a weekly job to run these tests?